### PR TITLE
No useless return

### PR DESCRIPTION
**Pull request by MathiasReker**
There should not be an empty return statement at the end of a function.

### CODE REVIEW

File: releases/8.0/common.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
 function language_redirect(string $currentLang): void {
     // We don't use the general language selection of php.net,
     // so soldier on with this one.
-    return;
+
```
Comment (kamil-tekiela) [inline releases/8.0/common.php:9]: Please remove blank lines

File: releases/8.1/common.php [hunk 2]
```diff
@@ -7,7 +7,7 @@
 function language_redirect(string $currentLang): void {
     // We don't use the general language selection of php.net,
     // so soldier on with this one.
-    return;
+
```
Comment (MathiasReker) [inline releases/8.1/common.php:10]:
```suggestion
```

File: releases/8.0/common.php (hunk 1 above)
Comment (MathiasReker) [reply releases/8.0/common.php:9]: Fixed

**Comment by cmb69** [comment]
Shouldn't we better remove these functions and their calls altogether?

**Comment by localheinz** [comment]
For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.8.0/doc/rules/return_notation/no_useless_return.rst.

**Comment by kamil-tekiela** [comment]
Obsolete due to https://github.com/php/web-php/pull/562

**Pull request closed without merging**